hokusai/staging.yml

```diff
@@ -159,6 +159,26 @@ spec:
     - 104.16.0.0/12
     - 172.64.0.0/13
     - 131.0.72.0/22
+
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: horizon
+  annotations:
+    nginx.ingress.kubernetes.io/whitelist-source-range: 173.245.48.0/20,103.21.244.0/22,103.22.200.0/22,103.31.4.0/22,141.101.64.0/18,108.162.192.0/18,190.93.240.0/20,188.114.96.0/20,197.234.240.0/22,198.41.128.0/17,162.158.0.0/15,104.16.0.0/12,172.64.0.0/13,131.0.72.0/22
+    nginx.ingress.kubernetes.io/proxy-send-timeout: 3600
+    nginx.ingress.kubernetes.io/proxy-read-timeout: 3600
+spec:
+  rules:
+    host: releases-staging.artsy.net
```

Comment:
Not blocking but something to think about -- if we include dns/subdomain in the config rather than as a string we may be able to declare subdomains systematically (`serviceName + -staging`), may help us to standardize naming conventions.

Reply:
Nice idea!

Reply:
Addressed in https://github.com/artsy/artsy-hokusai-templates/pull/21 but rendered out to a string on project setup as some projects like gravity don't map to their project names